.github/workflows/codeql.yml

```
@@ -0,0 +1,43 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: '35 22 * * 5'
```

```suggestion
on:
  push:
    branches: [ "master" ]
  pull_request:
```

there's no reason to `cron` it if we're doing it on all pushes and pulls imo

---

hmm, I think maybe we keep the `cron` (incase if there is no activity for some time, the scan can still run and maybe find new issues if there are updates in the rules the scanner runs). `cron` will just run once a week

---

Ah good point!! Sounds good then